renovate/src/Renovate/BinaryFormat/ELF.hs

```diff
@@ -893,8 +893,13 @@ instrumentTextSection cfg hdlAlloc loadedBinary textAddrRange@(textSectionStartA
     let symbolicBlockMap = Map.fromList [ (RE.basicBlockAddress cb, sb)
                                         | (cb, sb) <- baseSymBlocks
                                         ]
+    newCodeAddr <- fromIntegral <$> R.asks reSegmentVirtualAddress
+    newCodeSize <- fromIntegral <$> R.asks reSegmentMaximumSize
```

Comment:
I _think_ these should never overflow because `Memword (ArchAddrWidth arch) ~ ElfWordType (ArchAddrWidth arch)`. Does that sound realistic @travitch?

Reply:
Yes, that should be true